disputer/index.js

```diff
@@ -51,6 +54,24 @@ async function run(address, shouldPoll, pollingDelay, priceFeedConfig) {
     const gasEstimator = new GasEstimator(Logger);
     const disputer = new Disputer(Logger, empClient, gasEstimator, priceFeed, accounts[0]);
 
+    // The EMP requires approval to transfer the disputer's collateral tokens in order to dispute
+    // a liquidation. We'll set this once to the max value using the Token client.
+    const tokenClient = new TokenBalanceClient(
+      Logger,
+      ExpandedERC20.abi,
+      web3,
+      await emp.collateralCurrency(),
+      await emp.tokenCurrency()
+    );
+    const collateralApprovalTx = await tokenClient.collateralToken.methods
```

Comment:
nit: should we check to see if it's already approved so we don't send the transaction if we don't have to? Maybe we could check if the current approval is > UINT_MAX / 2 and only call `approve` if it's not?

Reply:
I was thinking about that too and that's a reasonable threshold. done